templates/monorepo/lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "0.0.0",
+  "registry": "https://registry.npmjs.org/",
+  "publishConfig": {
+    "access": "public"
```

Comment:
Hey 👋, just stumbled across this PR and thought I'd add a couple of thoughts. Hope you don't mind! 

I wonder if it'd make sense to add a section to the docs letting consumers know that if they want to publish private packages, they'll need to change this.

Reply:
All the TSDX templates currently default to public and MIT license, so that would be independent of this PR